config/locales/fr.yml

```diff
@@ -39,6 +39,30 @@ fr:
     posts: Statuts
     remote_follow: Suivre à distance
     unfollow: Ne plus suivre
+  admin:
+    settings:
+      click_to_edit: Cliquez pour éditer
+      contact_information:
+        email: Entrez une adresse courriel publique
```

Comment:
My guess is that at some we'll have a dedicated system to translate locales (transifex, weblate, pootle, others...) so the big decisions like this one will be taken by people who know the subject there. :)